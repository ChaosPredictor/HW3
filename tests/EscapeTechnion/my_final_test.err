Error: The email already exists in the system
Error: The email already exists in the system
Error: The email already exists in the system
Error: The email entered does not belong to any company in the system
Error: The room ID already exists in the faculty
Error: The email entered does not belong to any company in the system
Error: The company or room cannot be removed because a reservation currently exists
Error: The company or room cannot be removed because a reservation currently exists
Error: The room is not available in the given time
Error: The room is not available in the given time
Error: The room is not available in the given time
Error: The email already exists in the system
Error: The email already exists in the system
Error: The email already exists in the system
Error: The room is not available in the given time
Error: The client already has an order made for this time
Error: The email entered does not belong to any client in the system
Error: The room is not available in the given time
