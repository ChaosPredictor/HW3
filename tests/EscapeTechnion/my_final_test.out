Day #0: 0 events in total
That is all for day #0!

The system has 18 faculties and in 1 days, they have made a revenue of 0
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!
One of the best faculties - Mechanical Engineering, has made a revenue of 0 so far!
One of the best faculties - Electrical Engineering, has made a revenue of 0 so far!

Day #1: 1 events in total
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of civil1@ from Civil and Environmental Engineering, for the time 6:00
	The room is best for a skill level of 7. 1 people came for the total price of 75
That is all for day #1!

Day #2: 4 events in total
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of civil1@ from Civil and Environmental Engineering, for the time 6:00
	The room is best for a skill level of 7. 5 people came for the total price of 375
The order was made by ecivil5@, from the faculty of Civil and Environmental Engineering, who is skilled at 6
	It was made for room 5, of mech1@ from Mechanical Engineering, for the time 6:00
	The room is best for a skill level of 5. 10 people came for the total price of 1000
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of civil1@ from Civil and Environmental Engineering, for the time 7:00
	The room is best for a skill level of 7. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of civil1@ from Civil and Environmental Engineering, for the time 8:00
	The room is best for a skill level of 7. 3 people came for the total price of 225
That is all for day #2!

Day #3: 0 events in total
That is all for day #3!

The system has 18 faculties and in 4 days, they have made a revenue of 2050
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 1050 so far!
One of the best faculties - Mechanical Engineering, has made a revenue of 1000 so far!
One of the best faculties - Electrical Engineering, has made a revenue of 0 so far!

Day #4: 2 events in total
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 6:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of mech1@ from Mechanical Engineering, for the time 7:00
	The room is best for a skill level of 1. 5 people came for the total price of 500
That is all for day #4!

The system has 18 faculties and in 5 days, they have made a revenue of 2925
One of the best faculties - Mechanical Engineering, has made a revenue of 1500 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 1425 so far!
One of the best faculties - Electrical Engineering, has made a revenue of 0 so far!

Day #5: 30 events in total
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of civil1@ from Civil and Environmental Engineering, for the time 6:00
	The room is best for a skill level of 7. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 21, of civil2@ from Civil and Environmental Engineering, for the time 6:00
	The room is best for a skill level of 7. 5 people came for the total price of 375
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 31, of civil3@ from Civil and Environmental Engineering, for the time 6:00
	The room is best for a skill level of 7. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 7:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 22, of civil2@ from Civil and Environmental Engineering, for the time 7:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 32, of civil3@ from Civil and Environmental Engineering, for the time 7:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 3, of civil1@ from Civil and Environmental Engineering, for the time 8:00
	The room is best for a skill level of 5. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 23, of civil2@ from Civil and Environmental Engineering, for the time 8:00
	The room is best for a skill level of 5. 5 people came for the total price of 375
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 33, of civil3@ from Civil and Environmental Engineering, for the time 8:00
	The room is best for a skill level of 5. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 4, of civil1@ from Civil and Environmental Engineering, for the time 9:00
	The room is best for a skill level of 4. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 24, of civil2@ from Civil and Environmental Engineering, for the time 9:00
	The room is best for a skill level of 4. 5 people came for the total price of 375
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 34, of civil3@ from Civil and Environmental Engineering, for the time 9:00
	The room is best for a skill level of 4. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 5, of civil1@ from Civil and Environmental Engineering, for the time 10:00
	The room is best for a skill level of 3. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 25, of civil2@ from Civil and Environmental Engineering, for the time 10:00
	The room is best for a skill level of 3. 5 people came for the total price of 375
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 35, of civil3@ from Civil and Environmental Engineering, for the time 10:00
	The room is best for a skill level of 3. 5 people came for the total price of 375
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 11:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 1, of mech1@ from Mechanical Engineering, for the time 11:00
	The room is best for a skill level of 1. 5 people came for the total price of 500
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 12:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 2, of mech1@ from Mechanical Engineering, for the time 12:00
	The room is best for a skill level of 2. 5 people came for the total price of 500
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 13:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 3, of mech1@ from Mechanical Engineering, for the time 13:00
	The room is best for a skill level of 3. 5 people came for the total price of 500
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 14:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 4, of mech1@ from Mechanical Engineering, for the time 14:00
	The room is best for a skill level of 4. 5 people came for the total price of 500
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 2, of civil1@ from Civil and Environmental Engineering, for the time 15:00
	The room is best for a skill level of 6. 5 people came for the total price of 375
The order was made by ecivil2@, from the faculty of Civil and Environmental Engineering, who is skilled at 9
	It was made for room 5, of mech1@ from Mechanical Engineering, for the time 15:00
	The room is best for a skill level of 5. 5 people came for the total price of 500
The order was made by ecivil5@, from the faculty of Civil and Environmental Engineering, who is skilled at 6
	It was made for room 5, of mech1@ from Mechanical Engineering, for the time 16:00
	The room is best for a skill level of 5. 5 people came for the total price of 500
The order was made by ecivil1@, from the faculty of Civil and Environmental Engineering, who is skilled at 10
	It was made for room 1, of mech1@ from Mechanical Engineering, for the time 17:00
	The room is best for a skill level of 1. 5 people came for the total price of 500
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 3, of mech1@ from Mechanical Engineering, for the time 17:00
	The room is best for a skill level of 3. 5 people came for the total price of 500
The order was made by ecivil4@, from the faculty of Civil and Environmental Engineering, who is skilled at 7
	It was made for room 4, of mech1@ from Mechanical Engineering, for the time 17:00
	The room is best for a skill level of 4. 5 people came for the total price of 500
The order was made by ecivil5@, from the faculty of Civil and Environmental Engineering, who is skilled at 6
	It was made for room 5, of mech1@ from Mechanical Engineering, for the time 17:00
	The room is best for a skill level of 5. 5 people came for the total price of 500
That is all for day #5!

Day #6: 0 events in total
That is all for day #6!

Day #1: 0 events in total
That is all for day #1!

Day #2: 0 events in total
That is all for day #2!

Day #3: 0 events in total
That is all for day #3!

Day #4: 1 events in total
The order was made by ecivil3@, from the faculty of Civil and Environmental Engineering, who is skilled at 8
	It was made for room 5, of mech1@ from Mechanical Engineering, for the time 6:00
	The room is best for a skill level of 5. 5 people came for the total price of 500
That is all for day #4!

